### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by NinjaBuilderPro**
Adds logs to the robot with information about the robot.

### CODE REVIEW

**Review by averwhy — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -36,13 +64,14 @@ public void robotInit() {
    *
    * <p>This runs after the mode specific periodic functions, but before LiveWindow and
    * SmartDashboard integrated updating.
-   */
+   */
```
Comment (averwhy) [inline src/main/java/frc/robot/Robot.java]: Unnecessary. Remove

**Review by averwhy — CHANGES_REQUESTED** [review]

**Review by aqhoang2005 — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (averwhy) [reply src/main/java/frc/robot/Robot.java]: image

**Review by averwhy — APPROVED** [review]

**Comment by averwhy** [comment]
LGTM 
image

**Review by aqhoang2005 — APPROVED** [review]
poggers

**Comment by jnallard** [comment]
Fixed #11

**Pull request merged**